config/samples/pulpproject_v1beta1_pulp_cr.ocp.ci.yaml

```diff
@@ -5,9 +5,8 @@ metadata:
 spec:
   deployment_type: pulp
   admin_password_secret: "example-pulp-admin-password"
-  ingress_type: route
+  ingress_type: nodeport
```

Comment:
Hum... since this is an OCP sample, I believe that keep using ocp routes would be better than nodeport.
What do you think?

Reply:
Makes sense.  I didn't put much thought about the context of the file, this edit was more of a practical move to make the docs example work.

So as an alternative, how you think we can do to make the getting started work?
Maybe use another sample file?